cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/utils/Utils.java

```diff
@@ -15,6 +15,8 @@
 
     private static String hostname = null;
 
+    private static String systemname = "cassandra-cluster";
```

Comment:
What is global configuration? You have configuration loaded from yaml. If user does not set it we should have something set here. Global configuration is not that global since it is part of core and passed around. Utils in commons is global.